lib/nav/page-list.vue

```diff
@@ -259,10 +286,10 @@
         this.isPopoverOpen = false;
       },
       selectSite(slug) {
-        const site = _.find(this.sites, s => s.slug === slug);
+        const site = _.find(this.sites, s => s.subsiteSlug === slug || s.slug === slug);
```

Comment:
I think we can create a function because is the same condition in 2 places

Reply:
fixed